parsers/US_AK_SAP.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timezone
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Response, Session
+from requests_html import HTMLSession
+
+from electricitymap.contrib.lib.models.event_lists import ProductionBreakdownList
+from electricitymap.contrib.lib.models.events import ProductionMix
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.lib.exceptions import ParserException
+
+PARSER_NAME = "US_AK_SAP.py"
+
+SOURCE = "seapahydro.org"
+DATA_URL = "https://seapahydro.org/scada"
+
+
+def get_value_by_id(res: Response, id) -> float:
+    return float(res.html.find(f"#{id}", first=True).text)
```

Comment:
In combination with the above calling the API directly means we would get a json object that we can convert to a dict directly and avoid using HTML parsing and scraping.